pkg/plugin/plugin.go

```diff
@@ -419,14 +428,17 @@ func (e *AutoscaleEnforcer) Filter(
 
 	var resultString string
 	if allowing {
-		resultString = "allowing"
+		resultString = "Allowing"
 	} else {
-		resultString = "rejecting"
+		resultString = "Rejecting"
 	}
 
-	klog.Infof(
-		"[autoscale-enforcer] Filter: %s %s pod %v in node %s: %s; %s",
-		resultString, kind, pName, nodeName, cpuMsg, memMsg,
+	logger.Info(
+		fmt.Sprintf("%s Pod", resultString),
```

Comment:
`resultString` is either "Allowing" or "Rejecting" — the result is already derivable from the verdict, but IMO it's useful to have a message that makes sense by itself, without having to look at the keys (possibly many of them) to tell what actually happened.

As written, this will either have a message of "Allowing Pod" or "Rejecting Pod". I'd like to keep the messages.

Cleaned up `resultString` so that now it's called `message` and skips the unnecessary `fmt.Sprintf` step.